shoop/front/middleware.py

```diff
@@ -88,6 +90,11 @@ def _set_timezone(self, request):
     def process_response(self, request, response):
         if hasattr(request, "basket") and request.basket.dirty:
             request.basket.save()
+
+        is_superuser = request.user and request.user.is_superuser
```

Comment:
Also, you shouldn't need to check for the truthiness of `request.user` in the first place; if no one is logged in, it's an `AnonymousUser`